lib/ansible/modules/cloud/vmware/vmware_guest_facts.py

```diff
@@ -202,7 +202,6 @@
 from ansible.module_utils.vmware_rest_client import VmwareRestClient
 try:
     from com.vmware.vapi.std_client import DynamicID
-    from com.vmware.cis.tagging_client import Tag, TagAssociation
     HAS_VCLOUD = True
```

Comment:
Should this become HAS_VSPHERE?  For consistency with: https://github.com/ansible/ansible/blob/fd54a8b4ba5bc64a424d7b58ad32b4f531e78f06/lib/ansible/module_utils/vmware_rest_client.py#L30-L32

Reply:
Thanks @jillr Done the changes